src/utils/graphql/queries/getApplicationNEW.query.ts

```diff
@@ -26,5 +26,17 @@ export default gql`
         }
       }
     }
+    applicationTriggerStates(condition: { serial: $serial }, first: 1) {
```

Comment:
would this not be just the 'trigger' field on application ?

Reply:
Yes, that's right, now that we aren't using a single hook to check that we can simply use the field from the entity. Good point. When this is all finished we should remove the `applicationTriggerStates` view: New Issue [Back-end#225](https://github.com/openmsupply/application-manager-server/issues/225)